### ISSUE TITLE
Codes for preprocessing MPI-INF-3DHP dataset

### ISSUE THREAD

**Issue by nies14** [Vegetebird/MHFormer#6]
Could you share the codes for preprocessing the MPI-INF-3DHP dataset?

**Comment by Vegetebird** [comment]
I'm sorry for the late reply.
I will release the code about 3DHP. You can also use other MPI-INF-3DHP code for preprocessing.

**Comment by nies14** [comment]
Can you give me some links for some repo that has MPI-INF-3DHP preprocessing code?

and which reorder index list u did use for preprocessing the MPI-INF-3DHP dataset?

**Comment by Vegetebird** [comment]
Hi~you can refer to [Anatomy3D](https://github.com/sunnychencool/Anatomy3D/issues/5), [PoseAug](https://github.com/jfzhang95/PoseAug), [HMR](https://github.com/akanazawa/hmr), [SPIN](https://github.com/nkolot/SPIN).

**Comment by nies14** [comment]
How many joints are used for testing MPI-INF-3DHP?

**Comment by alecda573** [comment]
@nies14  @Ha0Tang @Vegetebird anyone ever figure out how to process MPI and train this model on it as well as evaluate?

also does anyone know the protocol they evaluate this under

**Comment by nies14** [comment]
Hi @alecda573. You can check [Ray3D ](https://github.com/YxZhxn/Ray3D) or [PoseAug](https://github.com/jfzhang95/PoseAug)

**Comment by alecda573** [comment]
@nies14 @Vegetebird neither of those provide codes or explain what their train/test split is.

have you found any codes for training or explaining the evaluation?

**Comment by nies14** [comment]
[PoseAug](https://github.com/jfzhang95/PoseAug) tested their model on a cross dataset. They trained using Human3.6m using ground truth keypoints and tested their model on the test set of MPI-INF-3DHP. You can use this script (https://github.com/jfzhang95/PoseAug/blob/main/run_evaluate.py) and to use the AUC and PCK metric remove comments from here (https://github.com/jfzhang95/PoseAug/blob/f3f5c4e916ebf7529b873ec1c14c1ce0bf0f5cb1/function_poseaug/model_pos_eval.py#L70)


But this MHFormer repo is not using the cross scenario evaluation.

**Comment by alecda573** [comment]
@nies14 ah good to know, so MhFormer does actually train on MPI and then evaluates on the validation set? 

Have you used the code you just linked to try reproducing their results on MPI?

**Comment by nies14** [comment]
@alecda573

Yes.

I didn't reproduce the result of MHFormer. I used those codes to test mine.

**Comment by alecda573** [comment]
@nies14 are you currently setting up MPI to train MhFormer on?

have you been able to reproduce MhFormer results on CPN from scratch on H36M?

Would you be willing to share the evaluation 
Of MPI you got from training on H36M?

**Comment by nies14** [comment]
@alecda573  I'm not working on Mhformer and I didn't reproduce their results too.

I tested PoseAug and the results are almost similar what they mentioned in their paper.

**Comment by qiqiApink** [comment]
@nies14 PoseAug use 16 joints, but in MHFormer is 17. Did you find the preprocessing code that be used directly by MHFormer and has same format with h36m?

**Comment by anas-zafar** [comment]
@qiqiApink were you able to draft a script for cross evaluation scenario for Mhformer?

**Comment by qiqiApink** [comment]
@anas-zafar I used the code of [P-STMO](https://github.com/paTRICK-swk/P-STMO).

**Comment by anas-zafar** [comment]
@qiqiApink this one https://github.com/paTRICK-swk/P-STMO/blob/main/run.py ?

**Comment by MehwishG** [comment]
@Vegetebird , I am trying to train MHformer on MPI-INF-3DHP dataset. I have followed the code of [P-STMO](https://github.com/paTRICK-swk/P-STMO) but still getting a large MPJPE of 298. Can you please share the checkpoint for MPI dataset?

**Comment by GloryyrolG** [comment]
> @alecda573
> 
> [PoseAug](https://github.com/jfzhang95/PoseAug) tested their model on a cross dataset. They trained using Human3.6m and ground truth keypoints and then tested their model on the test set of MPI-INF-3DHP. You can use this script (https://github.com/jfzhang95/PoseAug/blob/main/run_evaluate.py) and to use the AUC and PCK metric remove comments from here (https://github.com/jfzhang95/PoseAug/blob/f3f5c4e916ebf7529b873ec1c14c1ce0bf0f5cb1/function_poseaug/model_pos_eval.py#L70)
> 
> But this MHFormer repo is not using the cross scenario evaluation.

Hi @nies14 , @Vegetebird , do you believe the results are actually trained on MPI-INF-3DHP instead of cross-dataset generalization? But the paper says it is cross-dataset generalization. Many thanks & regards:)
